code/modules/client/preferences/_preferences.dm

```diff
@@ -1649,8 +1661,25 @@ GLOBAL_LIST_INIT(name_adjustments, list())
 		if(is_misc_banned(parent.ckey, BAN_MISC_PUNISHMENT_CURSE))
 			ADD_TRAIT(character, TRAIT_PUNISHMENT_CURSE, TRAIT_BAN_PUNISHMENT)
 
-		if(patreon)
-			character.accent = selected_accent
+	if(pref_species.multiple_accents && length(pref_species.multiple_accents))
+		change_accent = TRUE
+	else
+		change_accent = FALSE
+
+	if(patreon)
+		character.accent = selected_accent
+	if(patreon && change_accent)
+		//If i don't force it here, patreon users won't be native speakers of these languages because of the accent bypass in the species code.
```

Comment:
what does this mean

Reply:
Yeah same but it would suck if you are paying money to the owner and they are left out of a mechanic everyone has acess to, anyways i changed it to be set on the species and has nothing to do with _preference anymore